<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KlnSystem</class>
 <widget class="QMainWindow" name="KlnSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KlnSystem - Cadastro Cliente</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEditNome">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="cursorPosition">
         <number>0</number>
        </property>
        <property name="placeholderText">
         <string>Nome</string>
        </property>
        <property name="cursorMoveStyle">
         <enum>Qt::VisualMoveStyle</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditCpf">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursorPosition">
         <number>0</number>
        </property>
        <property name="placeholderText">
         <string>Cpf</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditEndereco">
        <property name="placeholderText">
         <string>Endereço</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditTelefone">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>(73) 8888-8888</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditEmail">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>email@mail.com</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="maximumSize">
         <size>
          <width>515</width>
          <height>400</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Nome</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Cpf</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Endereço</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Telefone</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>E-mail</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btSalvar">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btEditar">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Editar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btExcluir">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Excluir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
